lib/screens/recording_screen.dart

```diff
@@ -0,0 +1,274 @@
+              ],
+            ),
+          ),
+          actions: <Widget>[
+            FlatButton(
+              child: Text('Cancel'),
+              onPressed: () {
+                Navigator.of(context).pop();
+              },
+            ),
+            FlatButton(
+              child: Text('Delete'),
+              onPressed: () {
+                Navigator.of(context).pop();
+                _cancelRecording();
+              },
+            ),
+          ],
+        );
+      },
+    );
+  }
+
+  void _navigateToListTranslation() {
+    // TODO : implement Navigation Route to Translation Screen
+    print("Recording is Done, Navigating to next page");
+  }
+
+  Widget _languageWidget(String source, String target) {
+    return Container(
+        padding: EdgeInsets.all(16.0),
+        decoration: const BoxDecoration(
+          border: Border(
+            bottom: BorderSide(width: 1.0, color: Color(0xFFB2B7B6)),
+          ),
+        ),
+        child: Row(children: [
+          Text(
+            source,
+            style: TextStyle(fontWeight: FontWeight.bold),
+          ),
+          Expanded(child: Icon(Icons.compare_arrows)),
+          Text(target, style: TextStyle(fontWeight: FontWeight.bold))
+        ]));
+  }
+
+  Widget _sentenceWidget(String text) {
+    return Padding(
+        padding: EdgeInsets.all(16.0),
+        child: Text(
+          text,
+          style: TextStyle(
+              fontSize: 20.0,
+              color: Theme.of(context).accentColor,
+              letterSpacing: 0.75,
+              fontWeight: FontWeight.bold),
+        ));
+  }
+
+  Widget _recorderWidget() {
+    Widget stopIcon = Icon(Icons.stop,
+        size: _recorderIconSize, color: Theme.of(context).accentColor);
+    Widget micIcon = Icon(Icons.mic,
+        size: _recorderIconSize, color: Theme.of(context).accentColor);
+
+    Widget recorder = GestureDetector(
+        onTap: () {
+          (_isRecording) ? _stopRecording() : _startRecording();
+        },
+        child: Container(
+            child: _isRecording ? stopIcon : micIcon,
+            padding: new EdgeInsets.all(10.0),
+            decoration: new BoxDecoration(
+                border: new Border.all(
+                    color: Theme.of(context).primaryColorDark, width: 1.0),
+                borderRadius: new BorderRadius.circular(80.0),
+                color: Theme.of(context).primaryColorLight,
+                boxShadow: [
+                  new BoxShadow(
+                      color: Theme.of(context).primaryColorDark,
+                      blurRadius: 8.0)
+                ])));
+
+    return recorder;
+  }
+
+  Widget _controlsWidget() {
+    return Container(
+        padding: EdgeInsets.all(16.0),
+        child: Row(mainAxisAlignment: MainAxisAlignment.spaceEvenly, children: [
+          IconButton(
+              icon: Icon(Icons.delete),
+              iconSize: _recorderIconSize / 2,
```

Comment:
Why not divide by 3?

Reply:
I still think you should update the magic number. It hides an assumption on the size of the icon that is clear only to you.